test/image/mocks/axes_exponentbase_linear.json

```diff
@@ -0,0 +1,217 @@
+            "titlefont": {
+                "color": "#1f77b4"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#1f77b4",
+            "tickfont": {
+                "color": "#1f77b4"
+            },
+            "linecolor": "#1f77b4"
+        },
+        "legend": {
+            "x": 1.1,
+            "y": 1,
+            "bordercolor": "#000",
+            "borderwidth": 1
+        },
+        "yaxis2": {
+            "title": "yaxis2 title",
+            "titlefont": {
+                "color": "#ff7f0e"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#ff7f0e",
+            "tickfont": {
+                "color": "#ff7f0e"
+            },
+            "linecolor": "#ff7f0e",
+            "anchor": "free",
+            "position": 0.15,
+            "overlaying": "y",
+            "exponentbase": 2,
+            "exponentformat": "SI"
+        },
+        "yaxis3": {
+            "title": "yaxis3 title",
+            "titlefont": {
+                "color": "#2ca02c"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#2ca02c",
+            "tickfont": {
+                "color": "#2ca02c"
+            },
+            "linecolor": "#2ca02c",
+            "anchor": "free",
+            "position": 0,
+            "overlaying": "y",
+            "exponentbase": 8,
+            "exponentformat": "B"
+        },
+        "yaxis4": {
+            "title": "yaxis4 title",
+            "titlefont": {
+                "color": "#d62728"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#d62728",
+            "tickfont": {
+                "color": "#d62728"
+            },
+            "showexponent": "last",
+            "linecolor": "#d62728",
+            "side": "right",
+            "overlaying": "y",
+            "exponentbase": 16,
+            "exonentformat": "SI"
+        },
+        "yaxis5": {
+            "title": "yaxis5 title",
+            "titlefont": {
+                "color": "#9467bd"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#9467bd",
+            "tickfont": {
+                "color": "#9467bd"
+            },
+            "exponentbase": 2,
+            "exponentformat": "e",
+            "showexponent": "first",
+            "linecolor": "#9467bd",
+            "anchor": "free",
+            "side": "right",
+            "position": 0.85,
+            "overlaying": "y"
+        },
+        "yaxis6": {
+            "title": "yaxis6 title",
+            "titlefont": {
+                "color": "#8c564b"
+            },
+            "showline": true,
+            "ticks": "outside",
+            "tickcolor": "#8c564b",
+            "tickfont": {
+                "color": "#8c564b"
+            },
+            "exponentbase": "11",
```

Comment:
Image tests shouldn't have invalid values in their json mocks (that's what jasmine defaults tests are for).